src/types.ts

```diff
@@ -25,6 +25,8 @@ export interface RequestOptions {
   readonly errorExpiryLength?: number;
   /** Poll with at least this frequency in miliseconds */
   readonly pollFrequency?: number;
+  /** If the result is stale then we suspend instead of using the cached value */
```

Comment:
Please don't mention suspend since that's implementation specific to useResource(). This marks the cache data as considered unusable not just out of date.